src/Features/VisualBasic/Portable/ExtractMethod/VisualBasicMethodExtractor.VisualBasicCodeGenerator.vb

```diff
@@ -154,6 +153,11 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.ExtractMethod
                 Return declStatement.Parent
             End Function
 
+            Protected NotOverridable Overrides Function GetOutermostCallSiteContainerToProcess(cancellationToken As CancellationToken) As SyntaxNode
+                Dim callSiteContainer = GetCallSiteContainerFromOutermostMoveInVariable(cancellationToken)
+                Return If(callSiteContainer, Me.SelectionResult.GetOutermostCallSiteContainerToProcess(cancellationToken))
+            End Function
```

Comment:
this is the 3 methods moved up to one.  with the differing behavior moved to vb's selection result type.